### DISCUSSION TITLE
Improve description for SEO; draft PR -- comments?

### DISCUSSION THREAD

**Discussion by danielfdickinson** [adityatelange/hugo-PaperMod#521] [Ideas]
I am sending as a draft PR (#497) as I think it would be awkward to paste this into a discussion or issue, and a purpose of draft PRs is discussion of a proposal that is known to need work.

According to Bing SEO scan meta description should be between 15 and 160
characters long. In addition, Bing complains if there is no meta
description. We therefore make a best effort at a useful description
meets these requirements, unless the user has explicity provided one.

Thoughts?

**Comment by adityatelange** [comment]
I think blog writers should take an effort to write a good description for their posts. 
A good search engine however does not rank pages based on the only description of a page.

The draft PR I see is too much of an effort to do for people who don't want to write a description, is what I feel.
Also, we have support for multilingual websites so we can't add few descriptor words there.

**Reply by danielfdickinson** [reply]
That makes sense. Thanks.
